.github/workflows/deploy.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
+    paths:
+      - "**.md"
```

Comment:
This should also include things like images, css and other `static/` resources

Reply:
My thinking was that if a new static resource is added, it must be referenced somewhere in the .md, right? Otherwise it's just part of the github repo.

Reply:
the resources could be edited on their own (e.g modify a diagram, or update the CSS)